clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1077,14 +1085,21 @@ static void buildArgTys(ASTContext &Context, CXXRecordDecl *KernelObj,
       PointeeTy = Context.getQualifiedType(PointeeTy.getUnqualifiedType(),
                                            Quals);
       QualType ModTy = Context.getPointerType(PointeeTy);
-      
+
       CreateAndAddPrmDsc(Fld, ModTy);
     } else if (ArgTy->isScalarType()) {
       CreateAndAddPrmDsc(Fld, ArgTy);
     } else {
       llvm_unreachable("Unsupported kernel parameter type");
+      return false;
```

Comment:
What happened here?  This seems like an unrelated change?

Reply:
It's unnecessary here, removed. Thanks.